client/src/components/ItemCard.tsx

```diff
@@ -56,12 +52,7 @@ export const ItemCard = ({ item, onClick }: Props) => {
 
   return (
     <div className={classes.card}>
```

Comment:
is this supposed to be `classNames={classes.card}` ? I noticed that Line 14 import `import classNames from "classnames";` is not being used

Reply:
ahh! good catch. I deleted the unused import.